docs/key_benefits.md

```diff
@@ -0,0 +1,26 @@
+# Key Benefits of `scores`
+
+## Data Handling
+
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+  - gridded Earth system data (e.g., numerical weather prediction models)
+  - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
+- Handles missing data, masking of data and weighting of results.
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+
+## Usability
+
+- A companion Jupyter Notebook [tutorial](project:./tutorials/Tutorial_Gallery.md) for each metric and statistical test that demonstrates its use in practice.
+- Novel scores not commonly found elsewhere (e.g., [ FIxed Risk Multicategorical (FIRM)](https://doi.org/10.1002/qj.4266), [Flip-Flop Index](https://doi.org/10.1002/met.1732)).
```

Comment:
Re. Flip-Flop Index link:
- Ideally I would link to the entry in the "included.md" table, but I didn't know how to do that.
- When linking to the paper references, I know we normally link to both the original Flip-Flop Index article, and the Circular Flip-Flop Index article - but I couldn't do that in a single link. Is it okay to just link to the Flip-Flop Index article (as is currently the case), or does anyone have an alternative suggestion?
- Or, should it instead link to the API entry?

FIRM link:
- Ideally I would link to the entry in the "included.md" table, but I didn't know how to do that.
- Should this link to the paper? 
- Or should it link to the API?